### PR TITLE
Replace the old debouncing with a cooldown period

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**

### CODE REVIEW

File: app/models/partition/script_runner.rb [hunk 1]
```diff
@@ -54,45 +57,25 @@ def status
...
-            # Wait out the minimum time period
-            elsif (diff = (finish_time - current_time).to_i) > 0
-              task.sleep diff
-            # Exit because the maximum time has been exceeded +/- one second
-            else
-              break
-            end
-          end
+        if interval < 0
+          # TODO: Change the configuration key
```
Comment (benarmston) [inline app/models/partition/script_runner.rb]: Is there a reason this can't be done now?  Also what needs changing here?

**Review by benarmston — CHANGES_REQUESTED** [review]
This looks good.  Just a comment about a `XXX` left in the code.

File: app/models/partition/script_runner.rb (hunk 1 above)
Comment (WilliamMcCumstie) [reply app/models/partition/script_runner.rb]: https://github.com/openflighthpc/flight-scheduler-controller/pull/52/commits/915ec90a091970ad3f06db625321f57f65af92af

**Comment by benarmston** [comment]
LGTM.

**Pull request merged**